app/_src/networking/cni.md

```diff
@@ -14,7 +14,7 @@ There are two options to do this with {{site.mesh_product_name}}:
 - use the standard `kuma-init`, which is the default
 - use the {{site.mesh_product_name}} CNI
 
-{{site.mesh_product_name}} CNI can be leveraged in the two installation methods for Kubernetes: using [`kumactl`](/docs/{{ page.version }}/installation/kubernetes) and with [Helm](/docs/{{ page.version }}/installation/helm).
+{{site.mesh_product_name}} CNI can be leveraged in the two installation methods for Kubernetes: using {% if_version lte:2.1.x %}[`kumactl`](/docs/{{ page.version }}/installation/kubernetes) and with [Helm](/docs/{{ page.version }}/installation/helm){% endif_version %}{% if_version gte:2.2.x %}[`kumactl`](/docs/{{ page.version }}/production/install-kumactl/) and with [Helm](/docs/{{ page.version }}/production/install-kumactl/){% endif_version %}.
```

Comment:
can these if/else statements be on multiple lines? not sure about others but it's really hard to figure out which goes into `<= 2.1` and which ones go to `> 2.1`